### DISCUSSION TITLE
Problems with remote miBoxer pairing (TS1002)

### DISCUSSION THREAD

**Discussion by pilou8830** [Koenkk/zigbee2mqtt#12945] [Devices]
hello.
I can't pair my miboxer zigbee 3.0 remote control
In Home assistant > Zigbee2mqtt Edge it tells me that the remote is not supported and the model is TS1002.
With Zigbee Builder: _TZ3000_xwh1e22x.
Maybe the name of the manufacturer has changed.

![miboxer](https://user-images.githubusercontent.com/48477947/175807508-430fdffb-5aab-43fb-9f0e-2ac6e411dac2.JPG)

maybe just add xx in zigbee2mqtt edge? because this remote control is already in the list of "supported devices"

![miboxer 1](https://user-images.githubusercontent.com/48477947/175808148-61780a57-29a5-473e-bd27-e2d62b2061cb.JPG)

thank you in advance

**Comment by Koenkk** [comment]
`_TZ3000_xwh1e22x` is supported in z2m 1.25.2

**Reply by pilou8830** [reply]
Thanks Koenk. I will do the update.

**Comment by pilou8830** [comment]
hello, z2m won't start.
The newspaper says: `6-overlay-suexec: fatal: can only run as pid 1`
how to check that I have the latest version of edge, because I have many devices not recognized under the stable version.
I can't find the version number.
I will try that, because the person has solved the problem :
upgrade path/procedure 1.18.1-1 -> newer releases

**Reply by Koenkk** [reply]
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Comment by pilou8830** [comment]
it works!
maybe I didn't understand.
I deleted the zigbee2mqtt repository then reloaded the repository with https://github.com/zigbee2mqtt/hassio-zigbee2mqtt.
edge installation for me and everything works even the remote.
thank you Koenkk
